README.md

```diff
@@ -611,3 +613,56 @@ status:
   binding:
     name: kafka-eftg9
 ```
+
+## Role-Based Access Control (RBAC)
+
+Kubernetes clusters often utilize Role-based access control (RBAC) to authorize subjects to perform specific actions on resources. When operating in a cluster with RBAC enabled, the service binding reconciler needs permission to read resources that provisioned a service and write resources that services are projected into. This extension defines a means for third-party CRD authors and cluster operators to expose resources to the service binding reconciler. Cluster operators **MAY** impose additional access controls beyond RBAC.
+
```

Comment:
Split the following bit into two `h3` sections, one for the `ClusterRole` definition for application and provisioned services and another section for reconciler implementations.  Move the examples under each of those to better differentiate which group they apply to.